test/cookbooks/verify-octo/recipes/tentacle.rb

```diff
@@ -18,10 +18,39 @@
 # limitations under the License.
 #
 
+tentacle = node['verify-octo']['tentacle']
+
 # Just make sure its not installed already
 octopus_deploy_tentacle 'Tentacle' do
   action :remove
-  version node['verify-octo']['tentacle']['version']
+  version tentacle['version']
+end
+
+# Install it here
+octopus_deploy_tentacle 'Tentacle' do
+  action :install
+  version tentacle['version']
+  checksum tentacle['checksum']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.